npm/src/controller/oauth.ts

```diff
@@ -449,6 +463,9 @@ export class OAuthController implements IOAuthController {
       // ignore error
     }
 
-    return rsp.claims;
+    return {
+      ...rsp.claims,
+      ...rsp.requested_params,
```

Comment:
shouldn't expand `requested` here, it should be wrapped inside `requested`

Reply:
Noted